client/src/uiKit/Button/Button.js

```diff
@@ -1,24 +1,76 @@
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import './Button.css';
 
-function Button({ text, variant, onClick }) {
+const Button = ({
+  text,
+  theme = 'primary',
+  variant = 'default',
+  icon,
+  className,
+  onClick,
+  large,
+  small,
+  left,
+  right,
+}) => {
   return (
-    <button type="button" className={classnames(variant)} onClick={onClick}>
-      {text}
+    <button
+      type="button"
+      className={classnames(
+        'btn',
+        className,
+        theme,
+        variant,
+        { '--istype': variant },
+        { '--large': large },
+        { '--small': small }
+      )}
+      onClick={onClick}
+    >
+      {icon && left && <span className="btn_icon">{icon}</span>}
+      {icon && !left && !right && <span className="btn_icon">{icon}</span>}
```

Comment:
Is two icons on the left of the text needed? You can just have `left` as a default prop